pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
Could you check your pnpm version? looks like you are using an older one. 7+ can be safely used

Reply:
Hmm, I installed them using ```npm run setup:project``` which runs ```npx pnpm@7.3.0 i && node scripts/setup-env-files.js && npx pnpm build```

So every dev should be using pnpm at version 7.3.0 when running this script, or?